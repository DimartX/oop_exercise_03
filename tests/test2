help
read
5 7 
1 2 
100 123
read
0 0
1 2
5 0
4 -2
figures
read 0.13 -0.13
1.13 1.87
5.13 -0.13
4.13
-2.13
figures
sum_area
read   
3.14 15.9
2.65 3.5
8.9 79.3
centers
sum_area
areas
delete 0
delete 3
figures
sum_areas
exit
