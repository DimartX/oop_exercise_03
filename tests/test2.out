read, delete, areas, centers, figures, sum_area, help, exit
Figures: 
Triangle: 
{5.000000 7.000000} 
{1.000000 2.000000} 
{100.000000 123.000000} 

Rectangle: 
{0.000000 0.000000} 
{1.000000 2.000000} 
{5.000000 0.000000} 
{4.000000 -2.000000} 

Figures: 
Triangle: 
{5.000000 7.000000} 
{1.000000 2.000000} 
{100.000000 123.000000} 

Rectangle: 
{0.000000 0.000000} 
{1.000000 2.000000} 
{5.000000 0.000000} 
{4.000000 -2.000000} 

Rectangle: 
{0.130000 -0.130000} 
{1.130000 1.870000} 
{5.130000 -0.130000} 
{4.130000 -2.130000} 

Summary area: 
21.500000
Centers: 
{35.333333 44.000000} 
{2.500000 0.000000} 
{2.630000 -0.130000} 
{4.896667 32.900000} 
Summary area: 
37.072500
Areas: 
1.500000
10.000000
10.000000
15.572500
There is no such element!
Figures: 
Triangle: 
{5.000000 7.000000} 
{1.000000 2.000000} 
{100.000000 123.000000} 

Rectangle: 
{0.000000 0.000000} 
{1.000000 2.000000} 
{5.000000 0.000000} 
{4.000000 -2.000000} 

Triangle: 
{3.140000 15.900000} 
{2.650000 3.500000} 
{8.900000 79.300000} 

Summary area: 
27.072500
